physics/GFS_rrtmg_pre.F90

```diff
@@ -637,13 +636,27 @@ subroutine GFS_rrtmg_pre_run (im, levs, lm, lmk, lmp, n_var_lndp, lextop,&
           enddo
         endif
 
+!>---   add smoke and dust ---
+       if (aero_dir_fdb) then
+         do k=1,lmk
+           do i=1,im
+             aer_nm(i,k,1 )=aer_nm(i,k,1 )+qgrs(i,k,ntdust)*0.33*1.e-9  ! dust bin1
```

Comment:
Are any of these hard-coded constants tunable? If so, consider making them variables.

Reply:
This is specifically designed to divide by 3 and do a unit conversion, so putting them in variables doesn't make much sense.

Reply:
This is tunable. The value was based on our 10-days retro runs. I will try to update it to variables.

Reply:
It's not a huge deal, I'm just averse to seeing hard-coded data be changed in-code in subsequent PRs, especially if folks reading the code have no idea what the numbers represent.